.drone.yml

```diff
@@ -141,7 +142,7 @@ services:
 
   selenium:
     group: build
-    image: selenium/standalone-chrome-debug:3.10.0
+    image: selenium/standalone-chrome:3.10.0
```

Comment:
i think we should leave debug for two reasons:
- only debug version contains vnc server inside
- debug version [has no problems with windows maximizing](https://github.com/SeleniumHQ/docker-selenium/issues/559#issuecomment-353565964) in most of cases